examples/sdk/react/src/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
+import { ErrorBoundary, BacktraceClient } from '@backtrace/react';
+import { Fallback } from './Fallback';
+
+const universe = 'your-universe';
```

Comment:
Can we make all our examples similar? can we expose it to consts like in the rest of our example and just store the submission URL? It's better to have only submission URL because this is what people will see in the integration guide

Reply:
Fixed